cypress/e2e/item/copy/listCopyMultiple.cy.js

```diff
@@ -118,16 +119,12 @@ describe('Copy items in List', () => {
       const { path: toItemPath } = SAMPLE_ITEMS.items[0];
       copyItems({ itemIds, toItemPath });
 
-      cy.wait('@copyItems').then(({ response: { body } }) => {
+      cy.wait('@copyItems').then(({ response: { statusCode } }) => {
         // check item is still existing in parent
         itemIds.forEach((id) => {
           cy.get(`${buildItemsTableRowIdAttribute(id)}`).should('exist');
         });
-        body.forEach((item) => {
-          cy.get(`${buildItemsTableRowIdAttribute(item.id)}`).should(
-            'not.exist',
-          );
-        });
```

Comment:
There was a typo in the setup code. We receive an error after the call to copyMultiple which mean there is no new items created in the response body. We were essentially checking that the items created were not present. Before the test was passing because the check did dot wait for the elements to be rendered.